frontend/src/composables/useRequestFee.ts

```diff
@@ -12,7 +12,12 @@ export function useRequestFee(
   const error = ref<string | undefined>(undefined);
   const amount = ref<number>(0);
 
-  const show = computed(() => !!provider.value && !!requestManagerAddress);
+  const show = computed(
+    () =>
+      !!provider.value &&
+      isSupportedChain(provider.value.chainId.value) &&
```

Comment:
Yes, you are right! The issue was is that we were checking using the reference of requestManagerAddress which was always true.. We should have used requestManageAddress.value instead! Anyway, in my opinion, I'll remove requestManagerAddress check and use isSupportedChain because it gives a more meaning for the check is about IMO!

Reply:
And this would include for sure enhancing the helper functions themeselves!

Reply:
A generic fix for the whole composables issues is gonna be provided!

Reply:
helper has been removed and code has been transferred into proper places (getters in stores or within `useWallet` composable)